lib/rules/no-misleading-character-class.js

```diff
@@ -190,7 +201,19 @@ module.exports = {
                     const flags = getStringIfConstant(flagsNode, scope);
 
                     if (typeof pattern === "string") {
-                        verify(node, pattern, flags || "");
+                        verify(node, pattern, flags || "", fixer => {
+                            if (node.arguments.length === 1) {
+                                return fixer.insertTextAfterRange(patternNode.range, ', "u"');
+                            }
```

Comment:
First argument may be parenthesised:

```js
new RegExp(("[👍]")); // regex is /[👍]/
```

In that case, the suggested fix wouldn't work as intended:

```js
new RegExp(("[👍]", "u")); // regex is /u/
```